src/osd/osd_types.cc

```diff
@@ -5085,7 +5085,7 @@ void ScrubMap::encode(bufferlist& bl) const
 
 void ScrubMap::decode(bufferlist::iterator& bl, int64_t pool)
 {
-  DECODE_START_LEGACY_COMPAT_LEN(3, 2, 2, bl);
+  DECODE_START_LEGACY_COMPAT_LEN(3, 3, 3, bl);
```

Comment:
It doesn't seem like it would need to be updated, it's handling the pool encoding change, not related to ordering? (Not that we shouldn't remove it, just not in this patch series hopefully)